src/lib/Components/ArtworkFilter/__tests__/ArtworkFiltersStore-tests.tsx

```diff
@@ -797,6 +797,31 @@ describe("Apply Filters", () => {
     })
   })
 
+  fit("applies false ways to buy filters correctly", () => {
```

Comment:
Woops I left this focused.

```suggestion
  it("applies false ways to buy filters correctly", () => {
```